server/chat-commands/info.ts

```diff
@@ -2471,6 +2471,10 @@ export const commands: ChatCommands = {
 			comment: comment,
 		});
 		this.sendReply(`You have requested to show the link: ${link}${comment ? ` (with the comment ${comment})` : ''}.`);
+		const notifyId = `${room.roomid}-rank-driver`;
+		let message = `|tempnotify|${notifyId}|Pending media request!`;
+		message += `|${user.name} has requested to show media in ${room.title}.|new media request`;
```

Comment:
```suggestion
		const message = `|tempnotify|pendingapprovals|Pending media request!` +
			`|${user.name} has requested to show media in ${room.title}.|new media request`;
```

You don't need the room ID in the notification ID; notifications are already scoped per-room.